### DISCUSSION TITLE
What about a version for iPad Pro ?

### DISCUSSION THREAD

**Discussion by Theremingenieur** [Sequel-Ace/Sequel-Ace#1122] [Ideas] [answered]
First of all, a huge "Thank you" to the developers of Sequel Ace! Just migrated from Sequel Pro and still being amazed that this great project is alive again!

As an IT freelancer, I use Sequel Ace (and formerly Pro) for mySQL database maintenance and to create customised data extractions for a few of my customers on demand. Working sometimes at unorthodox times and in unorthodox places, and doing lots of other IT tasks using my iPad Pro, I wonder if now, that macOS and iPadOS slowly converge (have already some iPad apps running on my M1 MacBook Pro), it wouldn't be a relatively simple task to recompile Sequel Ace to run in iPadOS ?

I'm not an experienced Apple developer, thus I have no clue what is needed to transform a Mac only App into a universal App, but I'd like to throw the question in the round.

Greetings from France
Thierry

**Comment by Jason-Morcos** [comment, marked as answer]
Unfortunately, as the ecosystem is now it would be incredibly difficult to port to iPad. Apple makes it really easy to port from iPad to Mac, but there's no similarly easy way to go the other way. It would be a significant re-write and months/years of development likely

**Reply by Theremingenieur** [reply]
Thank you for having taken the time for this exhaustive reply!
